Remove terminal in code-oss

This patch removes the entire terminal functionality from Code OSS, including all related services, views, editors, and integrations.
---
 .../terminal/browser/terminal.contribution.ts | 92 +------------------
 1 file changed, 2 insertions(+), 90 deletions(-)

Index: AWSCodeOSS/build/private/code-editor-src/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts
===================================================================
--- AWSCodeOSS.orig/build/private/code-editor-src/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts
+++ AWSCodeOSS/build/private/code-editor-src/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts
@@ -4,9 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { KeyCode, KeyMod } from '../../../../base/common/keyCodes.js';
-import { Schemas } from '../../../../base/common/network.js';
 import { isIOS, isWindows } from '../../../../base/common/platform.js';
-import { URI } from '../../../../base/common/uri.js';
 import './media/terminal.css';
 import './media/terminalVoice.css';
 import './media/widgets.css';
@@ -14,40 +12,26 @@ import './media/xterm.css';
 import * as nls from '../../../../nls.js';
 import { CONTEXT_ACCESSIBILITY_MODE_ENABLED } from '../../../../platform/accessibility/common/accessibility.js';
 import { ContextKeyExpr } from '../../../../platform/contextkey/common/contextkey.js';
-import { Extensions as DragAndDropExtensions, IDragAndDropContributionRegistry, IDraggedResourceEditorInput } from '../../../../platform/dnd/browser/dnd.js';
-import { SyncDescriptor } from '../../../../platform/instantiation/common/descriptors.js';
 import { InstantiationType, registerSingleton } from '../../../../platform/instantiation/common/extensions.js';
 import { Registry } from '../../../../platform/registry/common/platform.js';
 import { GeneralShellType, ITerminalLogService, WindowsShellType } from '../../../../platform/terminal/common/terminal.js';
 import { TerminalLogService } from '../../../../platform/terminal/common/terminalLogService.js';
 import { registerTerminalPlatformConfiguration } from '../../../../platform/terminal/common/terminalPlatformConfiguration.js';
-import { EditorPaneDescriptor, IEditorPaneRegistry } from '../../../browser/editor.js';
-import { ViewPaneContainer } from '../../../browser/parts/views/viewPaneContainer.js';
 import { WorkbenchPhase, registerWorkbenchContribution2 } from '../../../common/contributions.js';
-import { EditorExtensions, IEditorFactoryRegistry } from '../../../common/editor.js';
-import { IViewContainersRegistry, IViewsRegistry, Extensions as ViewContainerExtensions, ViewContainerLocation } from '../../../common/views.js';
 import { RemoteTerminalBackendContribution } from './remoteTerminalBackend.js';
-import { ITerminalConfigurationService, ITerminalEditorService, ITerminalGroupService, ITerminalInstanceService, ITerminalService, TerminalDataTransfers, terminalEditorId } from './terminal.js';
+import { ITerminalConfigurationService, ITerminalEditorService, ITerminalGroupService, ITerminalInstanceService, ITerminalService } from './terminal.js';
 import { registerTerminalActions } from './terminalActions.js';
-import { setupTerminalCommands } from './terminalCommands.js';
 import { TerminalConfigurationService } from './terminalConfigurationService.js';
-import { TerminalEditor } from './terminalEditor.js';
-import { TerminalEditorInput } from './terminalEditorInput.js';
-import { TerminalInputSerializer } from './terminalEditorSerializer.js';
 import { TerminalEditorService } from './terminalEditorService.js';
 import { TerminalGroupService } from './terminalGroupService.js';
-import { terminalViewIcon } from './terminalIcons.js';
 import { TerminalInstanceService } from './terminalInstanceService.js';
 import { TerminalMainContribution } from './terminalMainContribution.js';
-import { setupTerminalMenus } from './terminalMenus.js';
 import { TerminalProfileService } from './terminalProfileService.js';
 import { TerminalService } from './terminalService.js';
-import { TerminalViewPane } from './terminalView.js';
-import { ITerminalProfileService, TERMINAL_VIEW_ID, TerminalCommandId } from '../common/terminal.js';
+import { ITerminalProfileService, TerminalCommandId } from '../common/terminal.js';
 import { registerColors } from '../common/terminalColorRegistry.js';
 import { registerTerminalConfiguration } from '../common/terminalConfiguration.js';
 import { TerminalContextKeyStrings, TerminalContextKeys } from '../common/terminalContextKey.js';
-import { terminalStrings } from '../common/terminalStrings.js';
 import { registerSendSequenceKeybinding } from './terminalKeybindings.js';
 
 // Register services
@@ -68,68 +52,6 @@ registerWorkbenchContribution2(RemoteTer
 registerTerminalPlatformConfiguration();
 registerTerminalConfiguration();
 
-// Register editor/dnd contributions
-Registry.as<IEditorFactoryRegistry>(EditorExtensions.EditorFactory).registerEditorSerializer(TerminalEditorInput.ID, TerminalInputSerializer);
-Registry.as<IEditorPaneRegistry>(EditorExtensions.EditorPane).registerEditorPane(
-	EditorPaneDescriptor.create(
-		TerminalEditor,
-		terminalEditorId,
-		terminalStrings.terminal
-	),
-	[
-		new SyncDescriptor(TerminalEditorInput)
-	]
-);
-Registry.as<IDragAndDropContributionRegistry>(DragAndDropExtensions.DragAndDropContribution).register({
-	dataFormatKey: TerminalDataTransfers.Terminals,
-	getEditorInputs(data) {
-		const editors: IDraggedResourceEditorInput[] = [];
-		try {
-			const terminalEditors: string[] = JSON.parse(data);
-			for (const terminalEditor of terminalEditors) {
-				editors.push({ resource: URI.parse(terminalEditor) });
-			}
-		} catch (error) {
-			// Invalid transfer
-		}
-		return editors;
-	},
-	setData(resources, event) {
-		const terminalResources = resources.filter(({ resource }) => resource.scheme === Schemas.vscodeTerminal);
-		if (terminalResources.length) {
-			event.dataTransfer?.setData(TerminalDataTransfers.Terminals, JSON.stringify(terminalResources.map(({ resource }) => resource.toString())));
-		}
-	}
-});
-
-// Register views
-const VIEW_CONTAINER = Registry.as<IViewContainersRegistry>(ViewContainerExtensions.ViewContainersRegistry).registerViewContainer({
-	id: TERMINAL_VIEW_ID,
-	title: nls.localize2('terminal', "Terminal"),
-	icon: terminalViewIcon,
-	ctorDescriptor: new SyncDescriptor(ViewPaneContainer, [TERMINAL_VIEW_ID, { mergeViewWithContainerWhenSingleView: true }]),
-	storageId: TERMINAL_VIEW_ID,
-	hideIfEmpty: true,
-	order: 3,
-}, ViewContainerLocation.Panel, { doNotRegisterOpenCommand: true, isDefault: true });
-Registry.as<IViewsRegistry>(ViewContainerExtensions.ViewsRegistry).registerViews([{
-	id: TERMINAL_VIEW_ID,
-	name: nls.localize2('terminal', "Terminal"),
-	containerIcon: terminalViewIcon,
-	canToggleVisibility: false,
-	canMoveView: true,
-	ctorDescriptor: new SyncDescriptor(TerminalViewPane),
-	openCommandActionDescriptor: {
-		id: TerminalCommandId.Toggle,
-		mnemonicTitle: nls.localize({ key: 'miToggleIntegratedTerminal', comment: ['&& denotes a mnemonic'] }, "&&Terminal"),
-		keybindings: {
-			primary: KeyMod.CtrlCmd | KeyCode.Backquote,
-			mac: { primary: KeyMod.WinCtrl | KeyCode.Backquote }
-		},
-		order: 3
-	}
-}], VIEW_CONTAINER);
-
 // Register actions
 registerTerminalActions();
 
@@ -244,8 +166,4 @@ registerSendSequenceKeybinding('\u001f',
 	mac: { primary: KeyMod.WinCtrl | KeyCode.Slash }
 });
 
-setupTerminalCommands();
-
-setupTerminalMenus();
-
 registerColors();
Index: AWSCodeOSS/build/private/code-editor-src/src/vs/workbench/contrib/terminalContrib/quickAccess/browser/terminal.quickAccess.contribution.ts
===================================================================
--- AWSCodeOSS.orig/build/private/code-editor-src/src/vs/workbench/contrib/terminalContrib/quickAccess/browser/terminal.quickAccess.contribution.ts
+++ AWSCodeOSS/build/private/code-editor-src/src/vs/workbench/contrib/terminalContrib/quickAccess/browser/terminal.quickAccess.contribution.ts
@@ -27,10 +27,6 @@ quickAccessRegistry.registerQuickAccessP
 	placeholder: nls.localize('tasksQuickAccessPlaceholder', "Type the name of a terminal to open."),
 	helpEntries: [{ description: nls.localize('tasksQuickAccessHelp', "Show All Opened Terminals"), commandId: TerminalQuickAccessCommandId.QuickOpenTerm }]
 });
-const quickAccessNavigateNextInTerminalPickerId = 'workbench.action.quickOpenNavigateNextInTerminalPicker';
-CommandsRegistry.registerCommand({ id: quickAccessNavigateNextInTerminalPickerId, handler: getQuickNavigateHandler(quickAccessNavigateNextInTerminalPickerId, true) });
-const quickAccessNavigatePreviousInTerminalPickerId = 'workbench.action.quickOpenNavigatePreviousInTerminalPicker';
-CommandsRegistry.registerCommand({ id: quickAccessNavigatePreviousInTerminalPickerId, handler: getQuickNavigateHandler(quickAccessNavigatePreviousInTerminalPickerId, false) });
 
 registerTerminalAction({
 	id: TerminalQuickAccessCommandId.QuickOpenTerm,
